plan/logical_plans.go

```diff
@@ -244,7 +244,7 @@ func (p *LogicalApply) extractCorrelatedCols() []*expression.CorrelatedColumn {
 
 // LogicalExists checks if a query returns result.
 type LogicalExists struct {
-	baseLogicalPlan
+	logicalSchemaProducer
```

Comment:
Where we use it? And there is no mention of "Exists" in your description.

Reply:
logicalSchemaProducer is contained by plans which need produce schema by itself instead of it's child. `Exists` output a boolean column which only depends on itself, so it contains logicalSchemaProducer

Reply:
So please update your description.